app/src/components/sections/SingleTeamView/InvestorsHeader.vue

```diff
@@ -0,0 +1,50 @@
+<template>
+  <div class="flex justify-between">
+    <div class="flex flex-col text-right">
+      <h2>Balance</h2>
+      <p v-if="!loadingTokenBalance && !tokenSymbolLoading">
+        <span class="text-3xl">{{
+          tokenBalance != undefined ? formatEther(tokenBalance) : '-'
+        }}</span>
+        {{ tokenSymbol }}
+      </p>
+      <span
+        v-if="loadingTokenBalance && tokenSymbolLoading"
+        class="loading loading-dots loading-sm self-end"
+      ></span>
+    </div>
+    <div class="flex flex-col text-right gap-y-4">
+      <h3>Total Supply</h3>
+      <p v-if="!tokenSymbolLoading && !totalSupplyLoading">
+        <span class="text-3xl">{{
+          totalSupply != undefined ? formatEther(totalSupply) : '-'
+        }}</span>
+        {{ tokenSymbol }}
+      </p>
+      <span
+        v-if="tokenSymbolLoading && totalSupplyLoading"
+        class="loading loading-dots loading-sm self-end"
+      ></span>
+      <div class="flex gap-x-1">
+        <h4>Contract Address :</h4>
+        <AddressToolTip :address="team.investorsAddress ?? ''" />
+      </div>
+    </div>
+  </div>
+</template>
+
+<script setup lang="ts">
+import type { Team } from '@/types'
+import { formatEther } from 'viem'
+import AddressToolTip from '@/components/AddressToolTip.vue'
+
+defineProps<{
+  team: Team
+  tokenSymbol: string | undefined
+  totalSupply: bigint | undefined
+  tokenSymbolLoading: boolean
+  totalSupplyLoading: boolean
+  tokenBalance: bigint | undefined
+  loadingTokenBalance: boolean
+}>()
```

Comment:
```suggestion
defineProps<{
  team: Team
  tokenSymbol: string 
  totalSupply: bigint
  tokenSymbolLoading: boolean
  totalSupplyLoading: boolean
  tokenBalance: bigint 
  loadingTokenBalance: boolean
}>()
```

The token alway have a symbol, 
the total suply always have a value, from 0 to n
And the token balance always have a value. at least 0

Reply:
but for this it cannot be done since there is also loading state in the investor header